build/toolchain/managed_toolchain.bzl

```diff
@@ -1,5 +1,12 @@
 load(":cc_toolchain_config.bzl", "cc_toolchain_config")
 
+aarch64_glibc_distros = {
+    "rhel9": "11",
+    "rhel8": "8",
+    "aws2023": "11",
+    "aws2": "7",
```

Comment:
In the `matrix-commitly.yml` file, label `ubuntu-22.04-arm64` is added. Do we want to add ubuntu related configuration here?

Reply:
the addition in `matrix-commitly.yml` was a temporary test, i'm going to revert it now. the four addition in `managed_toolchain.bzl` was what we trying to add support by using our home-brewed toolchain from [here](https://github.com/Kong/crosstool-ng-actions).
it's because redhat distros doesn't provide cross compilers to build userspace programs 😢 
but debian-based distros does, we just need to `apt install crossbuild-essentials-arm64` and don't need to manage/download them within Bazel.